components/rmrk/Gallery/Holder/Holder.vue

```diff
@@ -326,11 +329,12 @@ export default class Holder extends mixins(ChainMixin, KeyboardEventsMixin) {
   }
 
   private percentageTextClassName(percentage: number) {
-    return percentage >= 1
-      ? 'has-text-success'
-      : percentage > 0
-      ? 'has-text-danger'
-      : ''
+    if (percentage >= 100) {
```

Comment:
FYI there is condition mismatch :) 

![Screenshot 2022-04-10 at 10 21 49](https://user-images.githubusercontent.com/22471030/162609506-cdf86f51-363f-42c4-9ff5-6aae9e6290de.png)

Reply:
Because you prefer to use `toPercent` instant of `percentageFilter`, so the source data need to be changed. 1 -> 100